website/src/pages/learn/metadata.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Music metadata
+layout: ../../layouts/MainLayout.astro
+---
+
+import { MiniRepl } from '../../docs/MiniRepl';
+import { JsDoc } from '../../docs/JsDoc';
+
+# Music metadata
+
+You can optionally add some music metadata in your Strudel code, by using tags in code comments like this:
+
+```
+// @license: CC BY-NC-SA 4.0 https://creativecommons.org/licenses/by-nc-sa/4.0/
```

Comment:
I wonder why it's with colons here, but the examples don't use colons?

Reply:
Just a mistake here, I will remove it.
I made colons optional, so the music title should be the same anyway.